packages/cli/src/tests/baseCommand.js

```diff
@@ -1,51 +1,78 @@
 const BaseCommand = require('../oclif/ZapierBaseCommand');
+const { flags } = require('@oclif/command');
 const { buildFlags } = require('../oclif/buildFlags');
 
 const should = require('should');
 const { stdout } = require('stdout-stderr');
 
-const MESSAGE = 'very cool message';
+const MESSAGE = 'Here is a very cool message';
 const ROWS = [{ id: 123, title: 'hello' }, { id: 456, title: 'world' }];
 const ROW_HEADERS = [['Contact ID', 'id'], ['Neat Title', 'title']];
 
-class TestSampleCommand extends BaseCommand {
+class TestBaseCommand extends BaseCommand {
+  throwForInvalidAppInstall() {}
+}
+// logs both text and a table, which behave differently baed on the format flag
+class TestSampleCommand extends TestBaseCommand {
   async perform() {
-    switch (this.args.data) {
-      case 'table':
-        this.log('Here is some data!');
-        this.logTable({ rows: ROWS, headers: ROW_HEADERS });
-        break;
-      default:
-        this.log(MESSAGE);
-        break;
+    this.log(MESSAGE);
+    if (!this.flags.skipTable) {
+      this.logTable({ rows: ROWS, headers: ROW_HEADERS });
     }
   }
 }
-TestSampleCommand.args = [
-  {
-    name: 'data'
+
+TestSampleCommand.flags = buildFlags({
+  opts: { format: true },
+  commandFlags: {
+    skipTable: flags.boolean({
+      char: 's'
+    })
   }
-];
-TestSampleCommand.flags = buildFlags({ opts: { format: true } });
+});
+
+// logs and doesn't populate the `format` flag
+class NoFormatCommand extends TestBaseCommand {
+  async perform() {
+    this.log(MESSAGE);
+  }
+}
+NoFormatCommand.flags = buildFlags();
 
 describe('BaseCommand', () => {
   describe('log', () => {
     beforeEach(() => {
+      // stdout.print = true;
```

Comment:
Does this commented line of code have any purpose? I suggest removing if not.

Reply:
it's helpful for testing, though it probably doesn't need to be included. Will pull!